### PR TITLE
Added Jenkinsfile

### PR DESCRIPTION
**Pull request by jhryniuk**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent ant
```
Comment (partikus) [inline Jenkinsfile]: why `ant`?

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+    stage('Pull from github') {
+        git url: 'https://github.com/PGSSoft/php-docker-skeleton'
```
Comment (partikus) [inline Jenkinsfile]: better is to use [checkout](https://jenkins.io/doc/pipeline/steps/workflow-scm-step/#checkout-general-scm)

File: Jenkinsfile [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    stage('Run Tests') {
+        parallel php56: {
+            sh './docker.sh build-56';
+        }, php70: {
+            sh './docker.sh build-7';
+        }, php71: {
+            sh './docker.sh build';
+        },
+        failFast: true|false
```
Comment (partikus) [inline Jenkinsfile]: `failFast: true|false` what exactly this line is doing?

File: Jenkinsfile [hunk 4]
```diff
@@ -0,0 +1,29 @@
...
+    }
+    stage('Code Coverage') {
```
Comment (partikus) [inline Jenkinsfile]: IMHO does not make sense running tests twice (once without coverage and second one with coverage)

File: Jenkinsfile [hunk 5]
```diff
@@ -0,0 +1,29 @@
+pipeline {
+    agent ant
+    stage('Pull from github') {
+        git url: 'https://github.com/PGSSoft/php-docker-skeleton'
+    }
+    stage('Build images') {
+        sh './docker.sh build-images';
```
Comment (partikus) [inline Jenkinsfile]: building images could be run parallel

**Review by partikus — CHANGES_REQUESTED** [review]

**Comment by partikus** [comment]
@jhryniuk could you read my comments (unfortunately marked as `outdated`)?

**Comment by jhryniuk** [comment]
@partikus @dominikpgs CR please

**Comment by partikus** [comment]
@jhryniuk are you going to continue this PR ?

**Comment by jhryniuk** [comment]
I don't have idea how to solve this.

**Comment by partikus** [comment]
Closed in favor of #28

**Pull request closed without merging**